bundle-size/github-utils.js

```diff
@@ -164,6 +164,19 @@ class GitHubUtils {
     return false;
   }
 
+  /**
+   * Check whether the user is allowed to approve *all* bundle size change.
```

Comment:
Nit: /change/changes/s

Also, `*all*` makes me read this comment with jazz hands :open_hands:

Reply:
I think rather than emphasizing "all" here, it might be better to clarify in comments for `isBunderSizeApprover` that it only applies to...something. As I look closer and try to complete that sentence, it makes me think `isBundleSizeApprover` might be better named `isMemberOfTeam` or something similar.

Irrespective of naming, you could save/reuse a little bit of work here by sayng

```javascript
async isBundleSizeSuperApprover(username) {
  return await isBundleSizeApprover(username, process.env.SUPER_USER_TEAMS.split(',')); 
}
```

Reply:
Makes sense. Done.